### PR TITLE
#492 Sonar fixes

### PR DESCRIPTION
**Pull request by ghenzler**

### CODE REVIEW

File: accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ protected void doPost(final HttpServletRequest req, final HttpServletResponse re
 
         RequestParameters reqParams = RequestParameters.fromRequest(req, acInstallationService);
         LOG.info("Received POST request to apply AC Tool config with configurationRootPath={} basePaths={}",
-                reqParams.configurationRootPath, reqParams.basePaths);
+                escapeHtml4(reqParams.configurationRootPath), reqParams.basePaths);
```
Comment (kwin) [inline accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: why do you escape for HTML parsers in a log? A log line is never evaluated as HTML IMHO

File: accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ protected void doPost(final HttpServletRequest req, final HttpServletResponse re
         resp.setContentType("text/plain");
         if (((PersistableInstallationLogger) log).isSuccess()) {
             resp.setStatus(HttpServletResponse.SC_OK);
-            pw.println("Applied AC Tool config from " + reqParams.configurationRootPath + ":\n" + msg);
+            pw.println("Applied AC Tool config from " + escapeHtml4(reqParams.configurationRootPath) + ":\n" + msg);
```
Comment (kwin) [inline accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: This is plain text and therefore should not contain any HTML escaping!

**Review by kwin — CHANGES_REQUESTED** [review]
I think the Sonarcloud issues are rather false positives and should just be disabled with annotations (https://github.com/SonarSource/sonar-java/blob/2e3b7d25cebac51d5040d30dc3048c2f575d6e92/its/plugin/projects/suppress-warnings/src/main/java/SuppressWarningsAtMethodLevelWithRule.java#L4)

**Comment by kwin** [comment]
Still one FP: https://sonarcloud.io/project/issues?id=Netcentric_accesscontroltool&open=AXVCFo2RKPqjJP9f4pZJ&pullRequest=502&resolved=false&types=VULNERABILITY

Let us ignore javasecurity:S5145 as well.

**Comment by ghenzler** [comment]
@kwin For some reason Sonar still reports on `javasecurity:S5145` even thought it should be suppressed via 7e8ed5456c8cdf8b6debf5584eacf7dbc9cce5dc

**Pull request merged**